server/gqlgen.yml

```
@@ -4,7 +4,19 @@
 # for detailed .gqlgen.yml documentation.
 
 schema:
-  - schema.graphql
+  - ./gql/_shared.graphql
+  - ./gql/asset.graphql
+  - ./gql/user.graphql
+  - ./gql/team.graphql
+  - ./gql/project.graphql
+  - ./gql/scene.graphql
+  - ./gql/plugin.graphql
+  - ./gql/property.graphql
+  - ./gql/tag.graphql
+  - ./gql/cluster.graphql
+  - ./gql/was.graphql
+  - ./gql/layer.graphql
+  - ./gql/dataset.graphql
```

```suggestion
  - ./gql/*.graphql
```

https://github.com/99designs/gqlgen/issues/467

---

Done